src/silx/gui/widgets/RangeSlider.py

```diff
@@ -500,6 +505,14 @@ def setGroovePixmapFromProfile(self, profile, colormap=None):
         self.setGroovePixmap(qpixmap)
 
     # Handle interaction
+    def _mouseEventPosition(self, event):
+        if hasattr(event, "pos"):
+            # qt-5 returns QPoint
+            position = event.pos()
+        else:
+            # qt-6 returns QPointF
+            position = event.position()
```

Comment:
I think the following would be more coherent with existing code'. and from what I understood we should also use `position` for Pyside6
```suggestion
        if qt.BINDING in ('PyQt6', 'PySide6'):
            position = event.position()
            # qt-6 returns QPointF
        else:
            # qt-5 returns QPoint
            position = event.pos()
```

Reply:
I took the least invasive approach but I can change it.

However, testing for the binding instead of the presence of the method is not future proven.

If you want to use position for PyQt6 and PySide6, then one has to test for `position` and not for `pos`.

Reply:
If one wants to stick to using qt.BINDING, the test should be for using PyQt5 so that one is defaulting to the newer method.

Reply:
I agree but I think for the time being Qt5 is still the default / mostly used. I don't think we switched to Qt6 as the default Qt version yet.

Reply:
> I took the least invasive approach but I can change it.
> 
> However, testing for the binding instead of the presence of the method is not future proven.
> 
> If you want to use position for PyQt6 and PySide6, then one has to test for `position` and not for `pos`.

My point of view regarding the check of the biding is that you might be safer for future checking the binding.
Because if somehow this object get `pos` and `position` attribute then we would validate the condition anyway (I agree this is a highly unlikely ...)

Reply:
In the past there have been versions of PyQt/PySide with the capability to handle two different versions of Qt behind.

Personally if I have to use a method, I prefer to test for the presence of the method and not for the version.

@t20100 any opinion on this?